src/hash.cr

```diff
@@ -1,8 +1,37 @@
 require "crystal/hasher"
 
-# A `Hash` represents a mapping of keys to values.
+# A `Hash` represents a collection of key-value mappings, similar to a dictionary.
 #
-# See the [official docs](http://crystal-lang.org/docs/syntax_and_semantics/literals/hash.html) for the basics.
+# Main operations are storing a key-value mapping (`#[]=`) and
+# querying the value associated to a key (`#[]`). Key-value mappings can also be
+# deleted (`#delete`).
+# Keys are unique within a hash. When adding a key-value mapping with a key that
+# is already in use, the old value will be forgotten.
```

Comment:
```suggestion
# is already in use, the old value will be overwritten.
```

Reply:
I'm not sure, "overwritten" sounds like the actual value will be modified, but that's not the case. I think "forgotten" is fine.

Reply:
Maybe _replaced_ would do?